hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml

```diff
@@ -61,11 +67,22 @@
       <artifactId>protobuf-java</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
yet imported here. Why not just use hadoop-commons?

Reply:
Explicit addition is to support different version of hadoop-common in timelineserver. hadoop-common used in timelineserver may not have this dependency, yarn-api will need it. So explicit dependency is required (followed existing protobuf-java). As said above, to avoid enforce check failure due to SNAPSHOT version resolution, excluded this from hadoop-common above.
After this change, timelineserver tests started passing in both hbase1 and hbase2 profiles.